CNV/cnv-2.1.0.sh

```diff
@@ -14,7 +14,7 @@ TARGET_NAMESPACE="${TARGET_NAMESPACE:-openshift-cnv}"
 CLUSTER="${CLUSTER:-OPENSHIFT}"
 MARKETPLACE_NAMESPACE="${MARKETPLACE_NAMESPACE:-openshift-marketplace}"
 GLOBAL_NAMESPACE="${GLOBAL_NAMESPACE:-$globalNamespace}"
-CNV_VERSION="${CNV_VERSION:-2.1.0}"
+CNV_VERSION="${CNV_VERSION:-2.1.0-alpha}"
```

Comment:
Are people always going to be testing upgrade?  Will anyone be testing only deploy?

This value is configurable.  I would rather we keep `CNV_VERSION` pointed to latest instead of at a CSV that's not going to move very often.  Otherwise, we may confuse folks that are looking for bug fixes wondering where they are.

Reply:
makes sense